src/controllers/sqlDocumentService.ts

```diff
@@ -216,6 +220,21 @@ export default class SqlDocumentService implements vscode.Disposable {
         this._lastSavedUri = savedDocumentUri;
     }
 
+    private async onSuccessfulConnection(params: ConnectionSuccessfulEvent): Promise<void> {
+        const activeEditorKey = getUriKey(vscode.window.activeTextEditor?.document.uri);
+        const credentials = params?.connection?.credentials;
+        if (!credentials) {
+            return;
+        }
+        if (activeEditorKey === params.fileUri) {
+            // Always update when it's the active editor
```

Comment:
this covers the case where I connect a previously-unconnected .sql file that's open, right?  can you add that as a comment?  The current comment just tells me what the code is doing, but it'd be better if it tells the reader what the impact/significance of the line is.